code/style.css

```diff
@@ -1,7 +1,264 @@
+  justify-content: center;
+}
+
+.signupFormStyling {
+  background-color: #E7E7E7;
+  width: 60%;
+  display:flex;
+  justify-content: center;
+  border-radius: 10px;
+  box-shadow: 5px 5px 15px 0px #000000;
+}
+
+.signupFormGenderTitle {
+  margin-top: 10px;
+}
+
+.signupFormInterestsFlexbox {
+  display:flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+
+.signupFormButton {
+  margin-top: 10px;
+	box-shadow:inset 0px -3px 7px 0px #29bbff;
+	background:linear-gradient(to bottom, #2dabf9 5%, #0688fa 100%);
+	background-color:#2dabf9;
+	border-radius:3px;
+	border:1px solid #0b0e07;
+	display:inline-block;
+	cursor:pointer;
+	color:#ffffff;
+	font-family:Arial;
+	font-size:15px;
+	padding:9px 23px;
+	text-decoration:none;
+	text-shadow:0px 1px 0px #263666;
+}
+.signupFormButton:hover {
+	background:linear-gradient(to bottom, #0688fa 5%, #2dabf9 100%);
+	background-color:#0688fa;
+}
+.signupFormButton:active {
+	position:relative;
+	top:1px;
+}
+
+.accordionWrapper {
+  
+  display:flex;
+  justify-content: center;
+}
+.accordionContainer {
+  background-color: #E7E7E7;
+  width: 80%;
+  border-radius: 10px;
+  box-shadow: 5px 5px 15px 0px #000000;
+  width: 80%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  margin-bottom: 50px;
+}
+
+.accordionQuestion {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+.accordionParagraphContent {
+  display: none;
+}
+
+.accordionParagraphContentShown {
+  display: block;
+  padding-right: 20px;
+  padding-left: 20px;
+}
+
+.ham {
+  position: relative;
+  /* always keep ham on top of everything */
+  z-index: 1000;
+  width: 36px;
+  height: 36px;
+  border: none;
+  cursor: pointer;
+  background-color: transparent;
+  /* show the menu image */
+  background-image: url("https://ljc-dev.github.io/testing0/ham.svg");
+  background-size: 100%;
+}
+
+/* change ham image to close */
+.showClose {
+  background-image: url("https://ljc-dev.github.io/testing0/ham-close.svg");
+}
+
+.navbar {
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: black;
+  width: 100vw;
+  height: 50vh;
+  overflow: hidden;
+  color: white;
+  /* hide the menu above the screen by default */
+  transform: translateY(-100%);
+  /* transition adds a little animation when sliding in and out the menu */
+  transition: transform 0.2s ease;
+}
+
+.navbar ul {
+  width: 100%;
+  height: 100%;
+  list-style: none;
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.navbar ul li a {
+  color: white;
+  padding-bottom: 10px;
+  border-bottom: 2px solid black;
+  text-decoration: none;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 1.2rem;
+}
+
+.navbar ul li a:hover,
+.navbar ul li a:focus {
+  border-bottom: 2px solid white;
+}
+
+.showNav {
+  /* show the menu */
+  transform: translateY(0);
+}
+
+.TabletDesktopMenu {
+  display: none;
+}
+/* Tablet */
+@media (min-width: 1023px) {
```

Comment:
This comment is saying tablet, but I think this is desktop size 👀 Heads up: Your hero image takes up all height on desktop (and most of the height on tablet too):

<img width="949" alt="Skärmavbild 2021-02-04 kl  10 18 18" src="https://user-images.githubusercontent.com/51529201/106871230-4fa04180-66d2-11eb-909c-fcbbb9fa8c1e.png">